docs/details/shards/General/Drop.md

```diff
@@ -0,0 +1,13 @@
+`Drop` drops (removes) the last element of the sequence variable that has been passed to in the `:Name` parameter. 
+
+This shard works only on sequences and the parameters `:Key` and `:Global` are not applicable to it.
```

Comment:
Same remark: `:Key` and `:Global` have some use, we need to have samples for them.

Reply:
`:Global`

Similar behaviour as with `(Clear)` and `(Count)`.

Passing `(:Global true)` give error "Var is not a sequence"
```clj
[1 2 3] >= .seq
(Drop :Name .seq :Global true)
.seq (Log) ;;
;; => [error] [2022-07-27 18:06:10.880] [T-18768] [runtime.cpp::793] Shard activation error, failed shard: Drop, error: Variable is not a sequence, failed to Drop.
```

Passing `(:Global false)` works as expected, probably because it's equivalent to not passing `:Global` at all (default value is false)
```clj
[1 2 3] >= .seq
(Drop :Name .seq :Global false)
.seq (Log) ;; => [1, 2]
```

Reply:
`:Key`

`(Drop)` doesn't seem to work on tables. I get error "Var is not a sequence".

```clj
{:k1 1 :k2 [11 22]} >= .tbl
(Drop .tbl :Key "k1") (Log)
;; => [error] [2022-07-27 18:13:36.586] [T-23452] [runtime.cpp::793] Shard activation error, failed shard: Drop, error: Variable is not a sequence, failed to Drop. 
```

Reply:
Usage of `:Global` documented in commit [#0ecc5a77767aad041cd24a60afb1f15f83bc2a39](https://github.com/fragcolor-xyz/shards/pull/305/commits/0ecc5a77767aad041cd24a60afb1f15f83bc2a39)

Usage of `:Key` shall be documented in issue #335